<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_25" stretch="0">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QFrame" name="comment_area">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_48">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_78" stretch="1,5">
          <item>
           <widget class="QLabel" name="user_comment_icon_5">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_79">
            <item>
             <widget class="QLabel" name="comment_time_5">
              <property name="font">
               <font>
                <family>Segoe UI</family>
               </font>
              </property>
              <property name="text">
               <string>2025/04/23</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="comment_name_5">
              <property name="font">
               <font>
                <family>Segoe UI</family>
               </font>
              </property>
              <property name="text">
               <string>王曉明</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_80">
          <item>
           <widget class="QPushButton" name="like_button_5">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    icon-size: 24px;
}

QPushButton:hover {
    background-color: rgba(0, 0, 0, 30);  /* 半透明黑色遮罩 hover */
}


QPushButton:pressed {
    padding-left: 1px;
    padding-top: 1px;
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icons/icons/like.png</normaloff>:/icons/icons/like.png</iconset>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="like_number_5">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dislike_button_5">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    icon-size: 24px;
}

QPushButton:hover {
    background-color: rgba(0, 0, 0, 30);  /* 半透明黑色遮罩 hover */
}


QPushButton:pressed {
    padding-left: 1px;
    padding-top: 1px;
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/break-heart_icon.png</normaloff>icons/break-heart_icon.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dislike_number_5">
            <property name="font">
             <font>
              <family>Segoe UI</family>
             </font>
            </property>
            <property name="text">
             <string>33</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="comment_text_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
           </font>
          </property>
          <property name="text">
           <string>阿囉哈</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
